diff --git a/src/context/simulation_context.cpp b/src/context/simulation_context.cpp
index 69ec028..7e78af2 100644
--- a/src/context/simulation_context.cpp
+++ b/src/context/simulation_context.cpp
@@ -542,10 +542,7 @@ Simulation_context::print_info(std::ostream& out__) const
         }
         os << "maximum number of OMP threads : " << omp_get_max_threads() << std::endl
            << "number of MPI ranks per node  : " << mpi::num_ranks_per_node() << std::endl
-           << "page size (Kb)                : " << (get_page_size() >> 10) << std::endl
-           << "number of pages               : " << get_num_pages() << std::endl
-           << "total system memory (GB)      : " << (get_total_memory() >> 30) << std::endl
-           << "available memory (GB)         : " << (get_available_memory() >> 30);
+           << "page size (Kb)                : " << (get_page_size() >> 10) << std::endl;
         os << std::endl;
     }
     {
diff --git a/src/core/system_tools.hpp b/src/core/system_tools.hpp
index 4d8d46d..34a4411 100644
--- a/src/core/system_tools.hpp
+++ b/src/core/system_tools.hpp
@@ -40,24 +40,6 @@ hostname()
     return std::string(nm);
 }
 
-inline long
-get_page_size()
-{
-    return sysconf(_SC_PAGESIZE);
-}
-
-inline long
-get_num_pages()
-{
-    return sysconf(_SC_PHYS_PAGES);
-}
-
-inline long
-get_total_memory()
-{
-    return get_page_size() * get_num_pages();
-}
-
 inline auto
 get_proc_status()
 {
